addons/mail/static/src/components/message_list/message_list.xml

```diff
@@ -31,24 +31,24 @@
                     </div>
                 </t>
                 <!-- LOADING (if order asc)-->
-                <t t-if="threadView.threadCache.hasLoadingFailed">
+                <t t-if="messageListView.threadViewOwner.threadCache.hasLoadingFailed">
                     <div class="o_ThreadView_alertLoadingFailed alert alert-info">
                         <div>
                             An error occurred while fetching messages.
                         </div>
-                        <button class="o_ThreadView_alertLoadingFailedRetryButton btn btn-link" t-on-click="_onClickRetryLoadMoreMessages">
+                        <button class="o_MessageList_alertLoadingFailedRetryButton btn btn-link" t-on-click="_onClickRetryLoadMoreMessages">
                             Click here to retry
                         </button>
                     </div>
                 </t>
-                <t t-if="!threadView.threadCache.hasLoadingFailed and threadView.order === 'asc'">
+                <t t-if="!messageListView.threadViewOwner.threadCache.hasLoadingFailed and messageListView.threadViewOwner.order === 'asc'">
                     <t t-call="mail.MessageList.loadMore"/>
                 </t>
-                <div t-if="threadView.order === 'asc' and threadView.threadCache.orderedNonEmptyMessages.length !== 0 and !(threadView and threadView.thread and threadView.thread.model === 'mail.box')" class="flex-grow-1"/>
+                <div t-if="messageListView.threadViewOwner.order === 'asc' and messageListView.threadViewOwner.threadCache.orderedNonEmptyMessages.length !== 0 and !(messageListView.threadViewOwner and messageListView.threadViewOwner.thread and messageListView.threadViewOwner.thread.model === 'mail.box')" class="flex-grow-1"/>
```

Comment:
guard on `messageListView.threadViewOwner` existing is not necessary here, as it is already implied by the guard at the top of the template (was already useless before this PR)

Reply:
it's no big deal if you don't change, but just to note this was resolved but the change not done

Reply:
Oops, there was actually such an unused guard just below the `t-if="messageListView"`, and that's the one I removed. I just removed this one too. Sorry for omission!